include/openssl/dh.h

```diff
@@ -301,6 +306,13 @@ OPENSSL_EXPORT int i2d_DHparams(const DH *in, unsigned char **outp);
 OPENSSL_EXPORT int DH_compute_key(uint8_t *out, const BIGNUM *peers_key,
                                   DH *dh);
 
+// Standard parameters. These parameters are taken from RFC 5114.
+// This function returns a new DH object with standard parameters. It returns
+// NULL on allocation failure.
+// 
+// WARNING: Don't use this, this function has been deprecated and is only 
+// provided because to support MySQL.
+OPENSSL_EXPORT DH *DH_get_2048_256(void);
```

Comment:
Can you move this down to the deprecated section below line 917.
```suggestion
// Standard parameters. These parameters are taken from RFC 5114.
// This function returns a new DH object with standard parameters. It returns
// NULL on allocation failure.
// 
// WARNING: deprecated: Use .......
OPENSSL_EXPORT DH *DH_get_2048_256(void);
```
What is the alternative we should recommend customers to use? That is our standard pattern in header giles.